packages/web-react/src/components/Dialog/Dialog.tsx

```diff
@@ -7,7 +7,7 @@ import { useDialog } from './useDialog';
 // Solved using `as MutableRefObject<HTMLDialogElement | null>` but I do not like it
 
 const Dialog = (props: DialogProps, ref: ForwardedRef<HTMLDialogElement | null>): JSX.Element => {
-  const { children, isOpen, onClose, closeOnBackdropClick = true, ...restProps } = props;
+  const { children, isOpen, onClose, closeOnBackdropClick = true, closeOnEscapeKeyDown, ...restProps } = props;
```

Comment:
```suggestion
  const { children, isOpen, onClose, closeOnBackdropClick = true, closeOnEscapeKeyDown = true, ...restProps } = props;
```

Maybe if you use the default value, you shouldn't set the type by `as boolean` 🤔

Reply:
I wasn't sure where to set this default, so I decided to set it inside the `useCancelEvent` hook and not here, because it is just passing through this component.

Reply:
What about updating the `DialogProps` type?

Reply:
```diff
+  closeOnBackdropClick?: boolean;
+  closeOnEscapeKeyDown?: boolean;
```

Reply:
types updated